packages/utils/src/object.ts

```diff
@@ -92,50 +92,37 @@ export function urlEncode(object: { [key: string]: any }): string {
 }
 
 /**
- * Transforms any object into an object literal with all its attributes
- * attached to it.
+ * Transforms any `Error` or `Event` into a plain object with all of their enumerable properties, and some of their
+ * non-enumerable properties attached.
  *
  * @param value Initial source that we have to transform in order for it to be usable by the serializer
+ * @returns An Event or Error turned into an object - or the value argurment itself, when value is neither an Event nor
+ *  an Error.
  */
-export function convertToPlainObject(value: unknown): {
-  [key: string]: unknown;
-} {
-  let newObj = value as {
-    [key: string]: unknown;
-  };
-
+export function convertToPlainObject<V extends unknown>(value: V): Record<string, unknown> | V {
```

Comment:
we can maybe type this return values more strongly (and union type the various returns)

Reply:
Done: [9552a75](https://github.com/getsentry/sentry-javascript/pull/5047/commits/9552a755145c0e0aabd19e0c2c13b81dd698b6bb)